src/components/htmlComponent.ts

```diff
@@ -13,51 +17,42 @@ import * as RenderController from "../core/renderController";
 import * as Utils from "../utils";
 
 export class HTMLComponent extends AbstractComponent<HTMLElement> {
```

Comment:
still not sold on this separation between HTMLComponent and SVGComponent; the code's like 90% the same. IMO only the leaf components that actually render stuff (like a bar plot) should know anything about svg vs. html